lib/dom.js

```diff
@@ -541,6 +545,20 @@ declare class HTMLCollection<Elem: HTMLElement> {
   [index: number]: Elem;
 }
 
+// https://html.spec.whatwg.org/multipage/forms.html#category-listed
+type FormListedElement =
+  HTMLButtonElemnent |
+  HTMLFieldSetElement |
+  HTMLInputElement |
+  HTMLObjectElement |
+  HTMLOutputElement |
+  HTMLSelectElement |
+  HTMLTextAreaElement;
+
+declare class HTMLFormControlsCollection<Elem: FormListedElement> extends HTMLCollection<Elem> {
+  namedItem(name: string): RadioNodeList<Elem> | Elem;
```

Comment:
[This can be `null`](https://html.spec.whatwg.org/#dom-htmlformcontrolscollection-nameditem).

Reply:
fixed